### DISCUSSION TITLE
Badvatten i media

### DISCUSSION THREAD

**Discussion by salgo60** [salgo60/Svenskabadplatser#146] [Show and tell]
* Stockholms stads avrådan
  * SVT "[Efter regnovädret – stockholmare avråds från att bada](https://www.svt.se/nyheter/lokalt/stockholm/efter-regnovadret-stockholmare-avrads-fran-att-bada)" - 28 MAJ 2021
* Öresund – läckan lagad
  * "[Miljontals liter bajsvatten forsade ut i Öresund – läckan lagad](https://www.svt.se/nyheter/lokalt/skane/miljontals-liter-avloppsvatten-forsar-ut-i-oresund)"   1 JUNI 2021
* SVT [tweet från Hav](https://twitter.com/havochvatten/status/1400357542451089409?s=20) - 3 JUNI 2021
  * [Åsa Hoffman Stockholms miljöförvaltning](https://sverigesradio.se/artikel/7002443)
  * [Mikael Krysell](https://www.linkedin.com/in/mikaelkrysell/?originalSubdomain=se), Head of Environmental Monitoring Division at Havs- och vattenmyndigheten
  * Problem 
     * Öresunds vatten har problem
     * Stockholm avrådan pga regn och att avloppsvatten släppts ut i massor svar kommer imorgon 
     
![image](https://user-images.githubusercontent.com/14206509/120633681-b52af180-c46a-11eb-8bb5-9f482478f811.png)

* svt      
**Saknas media samma som**
se [inlägg om detta](https://community.dataportal.se/topic/87/hj%C3%A4lp-folk-att-bada-i-sommar-med-%C3%B6ppna-data-tips-och-hj%C3%A4lp-beh%C3%B6vs/61?_=1623230583384)
* på WD finns idag ämnes [topics från Nyheterna](https://w.wiki/3Ann)
![image](https://user-images.githubusercontent.com/14206509/120630789-6f205e80-c467-11eb-8a62-182752b6750c.png)

**Svenska media**
* [SPARQL](https://w.wiki/3RoG)
![image](https://user-images.githubusercontent.com/14206509/120631014-adb61900-c467-11eb-893f-2aa0f45a50c5.png)

**Comment by soleildeminuit** [comment]
![badvärdar](https://user-images.githubusercontent.com/10654578/123138313-2996f600-d455-11eb-86c4-6b572568eb5a.PNG)

[Badvärdar](https://www.upplands-bro.se/nyheter-2021/2021-06-23-upplands-bro-kommun---forst-i-sverige-med-badvardar-for-trygga-badplatser.html)

@salgo60 Kanske något att lägga till (platser/dagar/tider)?

**Comment by salgo60** [comment]
[Uppsala avråder](https://www.uppsala.se/kultur-och-fritid/aktiviteter-motion-och-idrott/motion-och-bad/badplatser/) snyggt att man även har det på [websidan för badet](https://www.uppsala.se/kultur-och-fritid/aktiviteter-motion-och-idrott/motion-och-bad/badplatser/lyssnaangsbadet/)

![image](https://user-images.githubusercontent.com/14206509/123207988-94304c00-d4be-11eb-8e17-0e2dbca59ed7.png)

* även Havs API har denna info se [badplatsen/api/detail/SE0210380000001171](https://badplatsen.havochvatten.se/badplatsen/api/detail/SE0210380000001171)

![image](https://user-images.githubusercontent.com/14206509/123208273-14ef4800-d4bf-11eb-8b35-3924b7244b5f.png)

* [badplatsen.havochvatten.se/badplatsen/karta/#/bath/SE0210380000001171](https://badplatsen.havochvatten.se/badplatsen/karta/#/bath/SE0210380000001171)

![image](https://user-images.githubusercontent.com/14206509/123208331-2d5f6280-d4bf-11eb-982a-c2ad8fef6fa9.png)
